packages/reports/addon/templates/components/metric-config.hbs

```diff
@@ -1,13 +1,13 @@
 {{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-{{#basic-dropdown
-  calculatePosition=calculatePosition
+<BasicDropdown
+  @calculatePosition={{calculatePosition}}
   as |dd|
-}}
-  {{#dd.trigger class="metric-config__dropdown-trigger" onMouseDown=(action "triggerFetch")}}
+>
+  <dd.trigger @class="metric-config__dropdown-trigger" @onMouseDown={{action "triggerFetch"}}>
```

Comment:
No @?
```suggestion
  <dd.trigger class="metric-config__dropdown-trigger" @onMouseDown={{action "triggerFetch"}}>
```

Reply:
Doesn't work without :man_shrugging:

Reply:
Yea just got this same situation in #594